### PR TITLE
[CI] Send slack message when changelog PR is ready

### PR DESCRIPTION
**Pull request by ThibaudAV**
## Issue

https://gravitee.atlassian.net/browse/APIM-367

## Description

Send slack message with new PR link

## Additional context
image

🚀 CI was able to deploy the build of this PR, so you can now try it directly [here](https://apimnightlywebui24386.z6.web.core.windows.net/ci-changelog-slack/index.html)
_Notes_: The deployed app is linked to the management API of APIM master. (Same login and password as APIM master)

---

📚&nbsp;&nbsp;View the storybook of this branch [here](https://612657caa8e859003a8a6430-ybvwfqguaa.chromatic.com)

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -1760,6 +1763,29 @@ jobs:
...
+        - when:
+              condition:
+                  not: << pipeline.parameters.dry_run>>
+              steps:
+                  - run:
+                        name: Get RELEASE_NOTES_PR_URL
+                        command: |
+                            echo "export RELEASE_NOTES_PR_URL=$(cat /tmp/releaseNotesPrUrl.txt)" >> $BASH_ENV
+                  - slack/notify:
+                        channel: C02NGT20S4W
```
Comment (gaetanmaisse) [inline .circleci/config.yml:1775]: Does it use a new channel? Or the one we are using for all release alerts?

Comment (ThibaudAV) [reply .circleci/config.yml:1775]: [#gravitee-release-alerts](https://graviteeio.slack.com/archives/C02NGT20S4W)

Comment (ThibaudAV) [reply .circleci/config.yml:1775]: I don't think a new channel is necessary 🤷‍♂️

**Review by phiz71 — APPROVED** [review]

File: .circleci/config.yml (hunk 1 above)
Comment (ThibaudAV) [reply .circleci/config.yml:1775]: seen with the @tech_writers_team 🚀 
> the release channel should be the best place to ping us from yes

p.s : new slack alias 🎸

**Pull request merged**